<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateCommitmentQWidget</class>
 <widget class="QWidget" name="CreateCommitmentQWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>475</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>475</width>
    <height>475</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create a Commitment</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="commitmentNameContainer" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="commitmentNameQLabel">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Commitment Name:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="commitmentNameQLineEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="commitmentDateContainer" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="QLabel" name="dateEndQLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>End Date</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QDateEdit" name="dateStartQDateEdit">
        <property name="displayFormat">
         <string>M/d/yyyy</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="dateStartQLabel">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Start Date</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QCheckBox" name="noEndDateQCheckBox">
        <property name="text">
         <string>No End Date</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDateEdit" name="dateEndQDateEdit">
        <property name="displayFormat">
         <string>M/d/yyyy</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="commitmentFrequencyContainer">
     <property name="title">
      <string>Frequency</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="3" column="1">
       <widget class="QComboBox" name="frequencyTimeQComboBox">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Minute(s)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hour(s)</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="qtyQLineEdit">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>1</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QComboBox" name="frequencyQComboBox">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <item>
         <property name="text">
          <string>Once a week</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Twice a week</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Three times a week</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Four times a week</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Five times a week</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Six times a week</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Everyday</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="QComboBox" name="commitmentModeQCombox">
        <item>
         <property name="text">
          <string>Weekly</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Everyday</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="commitmentBottomOptionsContainer" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="createCommitmentQPushButton_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>Create Commitment</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="backQPushButton_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
